hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.module.scss

```diff
@@ -0,0 +1,53 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.Cart {
+    display: flex;
+    color: color(backgroundC);
+
+    &Pic {
+        height: 90px;
+        width: 90px;
+    }
+
+    &Name {
+        color: color(font);
+        padding-bottom: 17px;
+    }
+
+    &ErrorName {
+        color: color(font);
+    }
+
+    &Error {
+        color: color(secondary);
+        padding-top: 0px;
+        padding-bottom: 1px;
+    }
+
+    &Parts {
+        display: flex;
+        flex-direction: column;
+        height: 90px;
+        width: 100%;
```

Comment:
Remove setting the height to a certain pixel (just remove the height attribute completely for now)